### PR TITLE
Update test cases for Redis DB

### PR DESCRIPTION
**Pull request by dinukaamarasinghe817**
## Purpose
To update the Redis DB test cases to include new code generation logic.

Related to: https://github.com/ballerina-platform/ballerina-library/issues/6216

## Checklist
- [x] Linked to an issue
- [ ] Updated the specification
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_100_redis/Config.toml [hunk 1]
```diff
@@ -1,2 +1,4 @@
 [persist_generate_100.entities.redis]
 connection = "redis://localhost:6379"
+
+
```
Comment (Nuvindu) [inline persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_100_redis/Config.toml:4]:
```suggestion
```

File: persist-cli/src/main/java/io/ballerina/persist/nodegenerator/syntax/clients/RedisClientSyntax.java [hunk 2]
```diff
@@ -332,15 +332,35 @@ private static String getJoinMetaData(Entity entity) {
             if (entityField.getRelation() != null) {
                 String relationType = BalSyntaxConstants.REDIS_ONE_TO_ONE;
                 Entity associatedEntity = entityField.getRelation().getAssocEntity();
+//                List<String> references = entityField.getRelation().getReferences();
```
Comment (Nuvindu) [inline persist-cli/src/main/java/io/ballerina/persist/nodegenerator/syntax/clients/RedisClientSyntax.java:335]: Is this line necessary?

Comment (dinukaamarasinghe817) [reply persist-cli/src/main/java/io/ballerina/persist/nodegenerator/syntax/clients/RedisClientSyntax.java]: No, we can remove that. I have removed it in https://github.com/ballerina-platform/persist-tools/pull/341/commits/06fbbe450e1f29e029b4f077d2da6c4a1e1bb9d8

File: persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_100_redis/Config.toml (hunk 1 above)
Comment (dinukaamarasinghe817) [reply persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_100_redis/Config.toml]: Fixed in https://github.com/ballerina-platform/persist-tools/pull/341/commits/06fbbe450e1f29e029b4f077d2da6c4a1e1bb9d8

**Review by daneshk — APPROVED** [review]
LGTM

**Pull request merged**